packages/financial-templates-lib/src/clients/ExpiringMultiPartyClient.js

```diff
@@ -138,6 +154,14 @@ class ExpiringMultiPartyClient {
       this.emp.methods.getCurrentTime().call()
     ]);
 
+    if (this.contractType == "Perpetual") {
```

Comment:
Set to the current `latestCumulativeFundingRateMultiplier` else default to zero. this makes other computations work as per usual with no changes needed in the non-perpetual case.